api/v1/Makefile

```diff
@@ -1,7 +1,14 @@
-include ../../images/hubble-proto/Makefile.defs
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+include ../../Makefile.defs
+
+# Update this via images/scripts/update-cilium-builder-image.sh
+CILIUM_BUILDER_IMAGE=quay.io/cilium/cilium-builder:3e04d507e2f7a0b569490060ddc8b9423c9fc8f0@sha256:7949a297fd2f07a008ef8a163ed84bfb338a37f3b36b68c2c9c5fddb324d325f
 
 .PHONY: proto
 proto:
-	docker container run --rm \
-		--volume $(CURDIR):/source \
-		cilium/hubble-proto:${version}
+	$(QUIET)$(CONTAINER_ENGINE) container run --rm \
+		--volume $(CURDIR):/src \
+		--user "$(shell id -u):$(shell id -g)" \
```

Comment:
Thanks for this, running as a regular user is definitely something we want (it was also part of my old draft PR #13405).